src/System.Linq/tests/ToSequenceTests.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public class ToSequenceTests
```

Comment:
The file and class should should probably be renamed to AsEnumerableTests (rather than ToSequenceTests), right?